applications/DKSAP.md

```diff
@@ -0,0 +1,117 @@
+
+Substrate EVM is an SRML module that provides an EVM execution environment for running unmodified Solidity code “natively” on a Substrate-based blockchain.Existing Solidity applications can be deployed and executed in this environment, and will gain the added benefits of being part of a Substrate-based blockchain.
+
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+Yahuang Wu
+
+* [Github address](https://github.com/wuyahuang)
+* 7 years of internet R&D experience, participated in the development of several APPs with millions of Daily Active Users (Qunar, Snowball, Meiyou)
+* Wrote 40 EOS technical articles [list of technical articles](https://github.com/meet-one/documentation/blob/master/meetone-lab-docs.md)
+* Selected for the EOS Open Source Community Acknowledgments List [list of selected lists](https://steemit.com/eos/@liondani/eos-acknowledgments-making-it-immutable-via-steemit )
+* Multiple EOS open-source repositories that submitted PRs were selected for the GitHub Archive Program
+* Has 10 blockchain technology patents [Patent List](http://www1.soopat.com/Home/Result?SearchWord=%E5%90%B4%E4%BA%9A%E7%9A%87&FMZL=Y&SYXX=Y&WGZL=Y&FMSQ=Y)
+
+### Contact
+
+- **Contact Name:** Yahuang Wu
+- **Contact Email:** wuyahuang@gmail.com
+
+### Legal Structure
+
+- **Registered Address:** 
+- **Registered Legal Entity:** 
+
+### Team's experience
+
+I have 7 years of experience in Internet research and development, focusing on the blockchain industry since 2018. I'm a deep participant in several technical communities, hackathon winners, and node service providers for EOS and PlatON.
+
+#### Team members' repos:
+
+- Yahuang Wu: [https://github.com/wuyahuang](https://github.com/wuyahuang)
+
+## Development Status :open_book:
+
+* [Faster Dual-Key Stealth Address for
+Blockchain-Based Internet of Things Systems](https://arxiv.org/pdf/1806.00951.pdf)
+
+* [PDKSAP : Perfected Double-Key Stealth Address Protocol without Temporary Key Leakage in Blockchain
+](https://ieeexplore.ieee.org/document/9209929)
+
+* [EDKSAP : Efficient Double-Key Stealth Address Protocol in Blockchain
+](https://ieeexplore.ieee.org/document/9724375)
+
+## Development Roadmap :nut_and_bolt:
+### Overview
+
+- **Total Estimated Duration:** 8 weeks
+- **Full-Time Equivalent (FTE):**  1
+- **Total Costs:** 10,000 USD
+
+### Milestone 1 — Node.js features
+
+- **Estimated duration:** 4 weeks
+- **FTE:**  1
+- **Costs:** 5,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | Apache 2.0 / GPLv3 / MIT / Unlicense |
+| 0b. | Documentation | We will provide both inline documentation of the code and a basic tutorial that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |
```

Comment:
I'm not sure what exactly you are proposing to deliver here. You mention Solidity in several places, but also Substrate nodes - is this meant to be a pallet? Solidity code? Solidity code which requires specific precompiles in Substrate? Note that we generally do not accept Solidity code as we are not experts in reviewing it. We would be interested in an ink! version, however.

The milestone deliverables for Milestone 1 and 2 should also be more specific - what specific features will be included and how will they be accessed? These will be used when we are verifying the deliverables, so more detail is generally better than less. If you look through some of the accepted grant PRs, you can see some examples of what is considered sufficient detail.